### PR TITLE
Final (!?) UI changes for new ArticlesPlus tab

### PR DESCRIPTION
**Pull request by akohler**
commit db3e1b0fbc2213944bd9e372ef5422cf44bba176
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Mon Mar 21 10:48:58 2016 -0800

    DRUP-813: Increment module versions

commit 503fa726a82591d1011d835535e6ccae6dde0bfb
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Mon Mar 21 10:41:41 2016 -0800

    DRUP-812: Add ArticlesPlus to search-log-data view, rename Articles to Articles (OLD)

commit 938ded196298ba1a7020159c3108a0c300077caa
Author: Andy Kohler <akohler@library.ucla.edu>
Date:   Mon Mar 21 10:30:37 2016 -0800

    DRUP-816: Remove ArticlesPlus image and theme template, replace with text description

### CODE REVIEW

**Comment by akohler** [comment]
@z3cka This is ready to go to www-test - please review/merge and deploy.

File: www/sites/all/modules/custom/ucla_tab_search/ucla_tab_search.module [hunk 1]
```diff
@@ -192,7 +190,16 @@ can be changed via a pulldown search limit on the results screen. Many records c
...
+    '#markup' => t('
+      <strong>What am I searching?</strong></br>
+      <a href="/#articlesplus">ArticlesPlus</a> searches many millions of scholarly articles,
+      newspaper articles, e-books, government documents, and more.  It combines hundreds of
+      online databases (like JSTOR and Web of Science) into a single search box.  See more
+      <a href="/about-articlesplus">about ArticlesPlus</a>.</br>
+      </br>
+      <a href="http://ucla.summon.serialssolutions.com/advanced">Advanced Search</a></br>
+      <a href="/search">Where else can I search?</a>
+    '),
```
Comment (z3cka) [inline www/sites/all/modules/custom/ucla_tab_search/ucla_tab_search.module]: as much as I hate stuffing markup into the array like this, this is the pattern that BS has used so I'll allow it. FWIW I'd like to see up use templates for most things like this.

**Comment by akohler** [comment]
Agreed - I though about keeping the theme/template created when this was an image, but decided that without that clear reason for a deviation, it was better to be consistent with the markup for all the other tabs.

**Comment by z3cka** [comment]
Agreed. Good choice. Merging now.

**Comment by z3cka** [comment]
deploying now.

**Comment by z3cka** [comment]
`Copying of code from here to www-test2.library.ucla.edu is complete.`

**Pull request merged**